backend/apps/matching/templates/participant/participant_dashboard.html

```diff
@@ -41,7 +41,11 @@
                 {% blocktrans %}Profile Details{% endblocktrans %}
             </div>
             <div class="card-body">
-
+                <label for="email">{% blocktrans %}E-Mail{% endblocktrans %}:</label>
+                <div class="">
+                    <input type="text" id="email" name="email" value="{{ request.user.email }}" class="textinput textInput form-control" disabled="">
+                    <small id="email_change_hint_text" class="form-text text-muted">{% blocktrans %}Your account is irreversibly linked to your email address. Should you wish to use another one, please delete your account and register with a new one.{% endblocktrans %}</small>
```

Comment:
I added a small helptext whose design is copied from the crispy help texts.
According to [this](https://developer.paciellogroup.com/blog/2012/01/html5-accessibility-chops-title-attribute-use-and-abuse/) website, using the `title`-attribute for hover texts is bad for accessibility and touch devices.
The result looks like this:
![Screenshot from 2020-07-12 00-15-55](https://user-images.githubusercontent.com/7229585/87234881-a6ac9500-c3d5-11ea-80e1-077e1b4370fa.png)

Reply:
I like it